common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2884,6 +2890,12 @@
     ],
     "sqlState" : "42809"
   },
+  "NOT_A_UNRESOLVED_ENCODER" : {
```

Comment:
"not an unresolved encoder" but how about just `NOT_UNRESOLVED_ENCODER`

Reply:
Thank you, It's better to use `NOT_UNRESOLVED_ENCODER` here, there are also similar names such as `NOT_NULL_CONSTRAINT_VIOLATION`.